lib/client/renderer.js

```diff
@@ -835,7 +839,8 @@ function init (client, d3) {
         .style('fill', 'white');
 
       label.append('text')
-        .style('font-size', 40 / opts.scale)
+        // reduce the treatment label font size to make it readable with SMB
+        .style('font-size', 25 / opts.scale)
```

Comment:
Feels very small with non-SMB user now; this would probably feel like a regression to most users. Wonder if we could something like count how many entries there are in the timeline and set the font size dynamically.

Reply:
That would be good.  I'm also playing around with getting rid of the 0. and U to make the label text narrower, and possibly removing the label entirely for the smallest boluses (maybe just 0.1U ones).

Reply:
#2926 has the leading 0 and trailing U removed, and changes the font size almost back up to where it was.

Reply:
Im not an open aps user (yet) but another interesting idea would be to display the bolus even at a different height in the chart based ok the IOB at the time.
Does that sound like a good idea? 
We generally don't care that much about the individual SMB but more about the totally IOB anyway I would think.
I'll try to experiment adding a different plugin that does that and maybe also draws the IOB curve as well to see how it looks. Would be a separate thing but hopefully it can help with the same problem.

Reply:
Oh, that's an interesting approach.  I have long liked the idea of rendering IOB and COB on the graph, as the NS Reports Day to day report does.  Plotting the insulin treatments along the IOB line, and the carb treatments along the COB line, would be a nice way to get them out of each other's way and make it easier to see everything (at the expense of cluttering things up a bit).